gulpfile.js

```diff
@@ -325,6 +335,20 @@ function release_win32() {
     return archive.finalize();
 }
 
+//Create distribution package for windows 64bit platform
+function release_win64() {
```

Comment:
Although my initial statement. You could optimize the code a little bit.
Rename this function to `release_win(arch)` and pass the `arch` parameter in the places where `'win64'` is used in this function.

Then call function `release_win(platform);`  from within the `case` statements.

The `release_win32()` function can be removed. This leaves one function to handle both code paths for `win32` and `win64`.

Reply:
Done in a second commit.